### DISCUSSION TITLE
PanV2 or V3 on Wyze Car Mod!

### DISCUSSION THREAD

**Discussion by gtxaspec** [gtxaspec/wz_mini_hacks#186] [General]
its nice.

What you need:

- V3 or modified PanV2 ( see #189 )
- micro usb otg cable, like one of the two below:
  - https://www.amazon.com/UGREEN-Adapter-Samsung-Controller-Smartphone/dp/B00LN3LQKQ
  -  https://www.amazon.com/Pack-USB-Port-Adapter-Cable/dp/B07B6CRW6D
- connect the otg cable to the camera, and the car into the bigger USB-A port
- connect your battery pack to the car 
- viewing:
  - enable RTSP in `wz_mini.config`, low_res preferred for latency reasons
    - connect via VLC, try setting network caching to a low value to get good latency, you may need to play around with this: `vlc rtsp://x.x.x.x:8554/video2_unicast --network-caching 0`
  - or use the official app to view!
- ssh into the camera
- run `car_control.sh` on the terminal and start driving!
- the official car app doesn't work with the v3 or panv2 obviously, so these are the methods to get this working! you can use the official app to move the pan around
- want a custom honking sound? replace `wz_mini/usr/share/audio/honk.wav` with a file of your choice.  needs to be at least 1 second long, 16bit, 16000hz, mono, wav format.
- this guy on youtube has some cool hardware mods: https://www.youtube.com/channel/UC2oF23aqoa3Zzl0OcDx1i-w/videos

**Comment by gtxaspec** [comment]
![20220714_142309-01.jpeg](https://user-images.githubusercontent.com/12115272/179091040-8523eb70-9ddd-4d2a-9566-f2b9aab35f80.jpeg)

**Reply by endertable** [reply]
Hehe, Pancam having the best seat in the house.
